installer/Product.wxs

```diff
@@ -229,7 +230,9 @@ SPDX-License-Identifier: MIT
 								ErrorControl="normal"
 								Account="[WIX_ACCOUNT_LOCALSERVICE]"
 								Vital="yes"
-								Interactive="no" />
+								Interactive="no">
+					<ServiceConfig ServiceSid="restricted" OnInstall="yes" OnReinstall="yes" />
```

Comment:
Why would we *not* use the Wix infrascructure, with automatic component management?

Reply:
By using `ebpfsvc.exe install` we will not need to duplicate the configurations at 2 places (like we are adding restricted service sid in wix installer, which is already done by install option in ebpfsvc), and possibly less chances of any future break.

Reply:
I'm not sure I see the benefits: [Windows Installer](https://learn.microsoft.com/en-us/windows/win32/msi/windows-installer-portal) (which WiX just leverages)) has a pretty clear format of well defining services, and gives you granular and transactional control on what to do in the different phases on the installation (i.e. install, repair, change, uninstall). 
All that would be gone using a cmd line, and imho would not make any difference even having an MSI in first place, instead of just a script, with limited component control.

One thing is when Windows Installer not supporting scenarios (i.e. installing driver services, etc.), but not using the native platform when available just waives all the component transition control.

I'm ultimately open to dismantle installing the service natively into a bare cmd invoke, if that is of concern to how we deliver our product.

Reply:
I prefer using the WIX framework as it's supposed to handle rollback in case of failures etc.